### ISSUE TITLE
Linters don't work in docker (master branch)

### ISSUE THREAD

**Issue by ivankorn** [terraform-google-modules/terraform-google-vpc-service-controls#7]
```
$ make docker_run
docker run --rm -it \
	-e PROJECT_ID \
	-e BUCKET_NAME \
	-e SERVICE_ACCOUNT_JSON \
	-e CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE=/cft/workdir/credentials.json \
	-e GOOGLE_APPLICATION_CREDENTIALS=/cft/workdir/credentials.json \
	-v /home/data/workspace/google/terraform-google-vpc-service-controls:/cft/workdir \
	gcr.io/cloud-foundation-cicd/cft/kitchen-terraform:1.0.1 \
	/bin/bash -c "source test/ci_integration.sh && setup_environment && exec /bin/bash"
Activated service account credentials for: [project-factory-9942@gl-ivankorniienko-seed.iam.gserviceaccount.com]
bash-4.4# make -s
Running shellcheck
xargs: shellcheck: No such file or directory
make: *** [Makefile:38: check_shell] Error 127
bash-4.4# 
```

**Comment by Tfmenard** [comment]
I don't think the linters are supposed to work with this image.
There is another image that has the linters installed.

**Comment by aaron-lane** [comment]
Fixed by #15.
